pages/contribute/github_way.md

```diff
@@ -17,15 +17,17 @@ The steps to be followed by contributors are detailed below.
 ### Announce and discuss your proposal through GitHub "issues"
 1. Go to the [RDM toolkit repo](https://github.com/elixir-europe/rdm-toolkit) on Github.
 2. Click on "Issues" in the top menu bar and check existing issues to see if your idea or suggestion is already being addressed.
-  * If yes, add your comments to the existing issue.
-  * If not, create a new issue by clicking on the green "New issue" on the right.
     {% include image.html file="new_issue_github.png" inline=true alt="Open new issue on GitHub." %}
-3. Choose one of the issues template and create a new issue. More information on creating issues can be found [here](https://docs.github.com/en/github/managing-your-work-on-github/creating-an-issue).
+2a. If an Issue exists
```

Comment:
this is sadly not working... will look for a fix

Reply:
@bedroesb what is not working? Do we need to hold this PR?

Reply:
@pinarpink The sentence below  " * If you will edit the content, **assign yourself to the issue** (at the right top)" is not technically feasible, because only people with write access can "assign things".

Reply:
@pinarpink the way of making a 2a sublist and making a sub sub list of bullets in combination with pictures renders wrongly